### PR TITLE
Merge develop into master 

### PR DESCRIPTION
**Pull request by clausmullie**
As per new #257

Checklist:
- [x] Spell checked
- [x] Correct Markdown and formatting
- [x] All hyperlinks work
- [x] All new files are linked in indices
- [x] Front matter is present and correct
- [x] Bullets and numbered lists are correct
- [x] All content reviewed

Being fixed with #255 

Additional items:
- [x] recruitment/hiring-process.md is not done - requires re-review  (Being fixed with #254)

### CODE REVIEW

File: activities/recruitment/hiring-process.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+## Processing reactions of people expressing interest:
+
+4. Questions about the content of the position: route to CEO for answering
+5. Questions about the procedure and the conditions: route to HRM for answering
+6. Applications are sent to HRM, for processing
+
+## Planned recruitment and hiring process:
+
+7. Current job openings have no closing date and applications are considered on a rolling basis. When we find a suitable candidate, we hire and close the opening. This means applications are considered ‘by themselves’ and not ‘in competition’ with a certain set of other applications.
+8. HRM confirms receipt to candidates and does first ‘sorting out’.
```
Comment (clausmullie) [inline activities/recruitment/hiring-process.md]: consider rephrasing 'sorting out' to round of evaluation

File: activities/recruitment/hiring-process.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+10. HRM notifies candidates (thank you or invitation for interview).
+11. First interview: CEO and HRM, 1 hr, partially on the basis of a pre-established list of desirable competences and other characteristics (i.e. fit for role, fit for organisation (culture, growth, leadership, long/short term)). Half of the interview is spent on presenting PublicCode.net, in the other half, the candidate has a change to present herself/himself. Immediately after the interview both HRM and CEO sore the candidate on the items on the list and determine if the candidate should progress to a second interview.
+12. Second interview: COO (same person as above) with HRM, also partially based on a pre-established list, and with attention to the particularities and matters not discussed in the first interview. Approximately. 80-90% of the time is spent on questioning the candidate. Immediately  after the interview both M and B/C score the candidate, based on the list and other impressions gathered. HRM takes this input to CEO and HRM and CEO determine if the candidate should receive an offer.
```
Comment (clausmullie) [inline activities/recruitment/hiring-process.md]: Cut "(same person as above)" as this it not correct
Replace M with HRM, B/C with CEO and COO

File: activities/recruitment/hiring-process.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+4. Questions about the content of the position: route to CEO for answering
+5. Questions about the procedure and the conditions: route to HRM for answering
+6. Applications are sent to HRM, for processing
+
+## Planned recruitment and hiring process:
+
+7. Current job openings have no closing date and applications are considered on a rolling basis. When we find a suitable candidate, we hire and close the opening. This means applications are considered ‘by themselves’ and not ‘in competition’ with a certain set of other applications.
+8. HRM confirms receipt to candidates and does first ‘sorting out’.
+9. HRM brings possibly suitable candidates to the attention of CEO en supervisor. These 2 make +/- judgement, without consulting each other. If judgement is not equal, a short discussion follows and judgement is made.
```
Comment (clausmullie) [inline activities/recruitment/hiring-process.md]: "en supervisor" is incorrect. 

Should this be "and COO"?

File: organization/roadmap.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+
+# Roadmap
+
+This roadmap serves as a model for how we're developing, growing and scaling our organization. It sets out the roles for both the staff and the members of the Foundation for Public Code.
+
+## Phase 0: Get the organization founded
+
+Staff of the Foundation for Public Code will:
+
+* [Build team](../roles/)
```
Comment (clausmullie) [inline organization/roadmap.md]: should this link not be (../roles/index.md) ?

File: activities/member-relations/user-mapping/index.md [hunk 5]
```diff
@@ -8,24 +8,22 @@ These are Foundation for Public Code value proposition hypotheses to test for us
...
 * [senior civil servants](senior-civil-servants.md)
 * [structural funding and philanthropy](structural-funding-philanthropy.md)
 
-### Not yet mapped
-
 Users and contributors:
 
 * the policy community
-* the tech and open source communities
+* [developers, designers](developers-and-designers.md) and the open source community
```
[the remark this answers is not in the document]
Comment (ElenaFdR) [reply activities/member-relations/user-mapping/index.md]:
```suggestion
* [developers, designers and the open source community](developers-and-designers.md) 
```

File: organization/roadmap.md (hunk 4 above)
Comment (ElenaFdR) [reply organization/roadmap.md]: Yes, but that's too big a change for this review. I've logged it.

File: organization/roadmap.md [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+* Develop a growth model for the organization
+
+This phase is over when members are able to commit to stage 3 and scaling.
+
+## Phase 3: Scale the organization
+
+Staff of the Foundation for Public Code will:
+
+* Provide codebase stewardship at scale
+* Grow our member base aggressively
```
Comment (ElenaFdR) [inline organization/roadmap.md]:
```suggestion
* Grow our member base proactively
```

**Comment by bvhme** [comment]
All of these comments are great but they don't belong in this pull request I feel, if we make these changes here then the release process becomes really hard. Problems identified here should be corrected in new pull requests to the `develop` branch and from there be pushed to master. If any of those things is broken than we can choose not to release until fixed.

But I guess you've talked about this already as well. I'd be eager to hear what you came up with.

**Comment by clausmullie** [comment]
Yup, developed a guide here #257

File: activities/member-relations/expression-of-interest.md [hunk 7]
```diff
@@ -15,7 +15,7 @@ You can access the [template here](https://docs.google.com/document/d/1jGQYAMkef
 ## Assumptions
 * This lightweight/low commitment Expression of Interest can be signed while we work with potential Founding Members on the Founding Membership Proposal
 * This resource is a link to a google doc for now, but might change format soon
-* Any changes to the content of this template must first be made in About before being propagated into this document
+* Any proposed changes to the content of this template must first be proposed and merged with [the `master` branch of About](https://github.com/publiccodenet/about/tree/master) before being implemented here
```
[the remark this answers is not in the document]
Comment (clausmullie) [reply activities/member-relations/expression-of-interest.md]: done in ca8ddfd256fb07a5f7008b837b73664cd73428db

File: activities/member-relations/founding-membership-proposal.md [hunk 8]
```diff
@@ -16,7 +16,7 @@ You can access the [template here](https://docs.google.com/document/d/1icscB-9jP
 * Our strategy is to work with our contacts within the organization to make a convincing proposal to their relevant level of political decision-making
 * We first send potential founding members this draft full proposal, which invites them to sign a light-weight/low-commitment Expression of Interest. This means we can talk about our working with them in the open, while continuing to tailor the details of this proposal
 * This resource is a link to a google doc for now, but might change format soon 
-* Any changes to the content of this template must first be made in About before being propagated into this document
+* Any proposed changes to the content of this template must first be proposed and merged with [the `master` branch of About](https://github.com/publiccodenet/about/tree/master) before being implemented here
```
[the remark this answers is not in the document]
Comment (clausmullie) [reply activities/member-relations/founding-membership-proposal.md]: fixed in #255

File: activities/member-relations/user-mapping/index.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (clausmullie) [reply activities/member-relations/user-mapping/index.md]: fixed in #255

File: activities/member-relations/user-mapping/index.md [hunk 9]
```diff
@@ -8,24 +8,22 @@ These are Foundation for Public Code value proposition hypotheses to test for us
...
-Other interested parties:
-
+Affiliates and partners:
 * vendors and the private sector
+* non-governmental organizations and non-profits aligned with our mission
+
+Other interested parties:
 * end users (residents and citizens)
-* journalists, academia and non-governmental organizations
+* journalists and academia and
```
[the remark this answers is not in the document]
Comment (clausmullie) [reply activities/member-relations/user-mapping/index.md]: fixed in #255

File: activities/recruitment/hiring-process.md (hunk 1 above)
Comment (clausmullie) [reply activities/recruitment/hiring-process.md]: fixed in #254

File: activities/recruitment/hiring-process.md (hunk 2 above)
Comment (clausmullie) [reply activities/recruitment/hiring-process.md]: fixed in #254

File: activities/recruitment/hiring-process.md (hunk 3 above)
Comment (clausmullie) [reply activities/recruitment/hiring-process.md]: addressed in #254

File: organization/roadmap.md (hunk 6 above)
Comment (clausmullie) [reply organization/roadmap.md]: fixed in #255

**Comment by bvhme** [comment]
Sooooo, perhaps we can close this PR until it is necessary again, now it just send notifications on every push which doesn't sound very useful.

**Pull request merged**